sfge/src/test/java/com/salesforce/graph/ops/ObjectFieldUtilTest.java

```diff
@@ -393,7 +393,8 @@ public void testRegroup_withMultipleDuplicatesInSingleObject() {
         assertThat(contactQueryInfo.isOutermost(), equalTo(true));
     }
 
-    // TODO: In the fullness of time, turn this test and the others like it into one parameterized test.
+    // TODO: In the fullness of time, turn this test and the others like it into one parameterized
```

Comment:
All of these changes are whitespace and formatting that got missed in the previous PR.